cmd/river/go.mod

```diff
@@ -22,9 +23,10 @@ require (
 	github.com/jackc/pgpassfile v1.0.0 // indirect
 	github.com/jackc/pgservicefile v0.0.0-20221227161230-091c0ba34f0a // indirect
 	github.com/jackc/puddle/v2 v2.2.1 // indirect
-	github.com/riverqueue/river/riverdriver v0.0.17 // indirect
+	github.com/oklog/ulid/v2 v2.1.0 // indirect
```

Comment:
seems like this is outdated diff?

Reply:
Ugh, I thought so too, but that's what `go mod tidy` produces. The ULID dep should be dropped next time we cut a release and can target the River CLI Go submodule against it.